build/pkg-scripts/postinstall

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+# This postinstall script is run on macOS pkg installers
+
+# Extract a referral / promo code from the installer path of the format /dir/path/Setup-Brave-Anything-xxx001.pkg
+# where the promo code would be xxx001
+installerPath=$1
+installerPathPromoCodeRegex='.+([a-zA-Z0-9]{3}[0-9]{3})\.pkg$'
```

Comment:
should this be `.+-([a-zA-Z0-9]{3}[0-9]{3})\.pkg$` to match the windows regex?

Reply:
got spot. `-` went missing during testing the different syntax